### ISSUE TITLE
Request to convert `Meteor/Number - DecimalJS` options

### ISSUE THREAD

**Issue by thearabbit** [zebra-lucky/meteor-aggregate-retyped#3]
I would like to request you the `Meteor/Number - DecimalJS` options

Ex:
```js
let doc = {
  amount : 123.456
}

doc = replaceTypes(doc, replaceMeteorAtomWithDecimalJS)
```
And support on `Client Code`

**Comment by zebra-lucky** [comment]
What do you mean by "support on Client Code"?

**Comment by thearabbit** [comment]
Can use it on `Client code`
```js
// client code
import {replaceType, ....} from .....
.............

let doc = {
  amount : 123.456
}

doc = replaceTypes(doc, replaceMeteorAtomWithDecimalJS)
Meteor.call('methodName', doc, (error, result)=>{
  .............
})
```

**Comment by thearabbit** [comment]
Could all of retype
- General Number <-> DecimalJS
- General Number <-> MongoDecimal
- DecimalJS <-> MongoDecimal

**Comment by zebra-lucky** [comment]
Minimongo on client does not know about Mongo.Decimal128 type

**Comment by thearabbit** [comment]
But I would like to convert normal document
```js
let doc = {
  amount : 123.456
  amount2: 6543.21
}

doc = replaceTypes(doc, replaceNumberAtomWithDecimalJS)
```
Instead of 
```js
let doc = {
  amount : Decimal(123.456)
  amount2: Decimal(6543.21)
}
```

**Comment by zebra-lucky** [comment]
Ok, I will add replaceNumberWithDecimal, replaceDecimalWithNumber?

**Comment by zebra-lucky** [comment]
published version 0.0.4

**Comment by thearabbit** [comment]
Very thanks again.
